### ISSUE TITLE
latest install AnnData is not defined

### ISSUE THREAD

**Issue by iandriver** [scverse/scanpy#416]
`pip install -e .`
gives:
```
Obtaining file:///apps/gau/scanpy
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/apps/gau/scanpy/setup.py", line 11, in <module>
        from scanpy import __author__, __email__
      File "/apps/gau/scanpy/scanpy/__init__.py", line 31, in <module>
        from . import tools as tl
      File "/apps/gau/scanpy/scanpy/tools/__init__.py", line 1, in <module>
        from ..preprocessing._simple import pca
      File "/apps/gau/scanpy/scanpy/preprocessing/__init__.py", line 7, in <module>
        from ._combat import combat
      File "/apps/gau/scanpy/scanpy/preprocessing/_combat.py", line 111, in <module>
        def combat(adata: AnnData, key: str = 'batch', inplace: bool = True):
    NameError: name 'AnnData' is not defined
```

**Comment by flying-sheep** [comment]
Fixed in de16b79f9cbf2bc881faa7a36ebf6aaeb24ee786, sorry!

**Comment by iandriver** [comment]
Thanks!
